BibBuddy/app/src/main/java/de/bibbuddy/TextNoteEditorFragment.java

```diff
@@ -22,6 +30,43 @@
   private Note note;
   private NoteModel noteModel;
 
+  @Override
+  public void onCreate(Bundle savedInstanceState)
+  {
+    super.onCreate (savedInstanceState);
+    requireActivity().getOnBackPressedDispatcher().addCallback(new OnBackPressedCallback(true) {
+      @Override
+      public void handleOnBackPressed() {
+        saveNote();
```

Comment:
The method for updating the note list for a book could be called here.

Reply:
On going back to BookNotesView the list is always fetched new from the database in onCreateView - there is no need to update it here, too.